components/_headers.scss

```diff
@@ -0,0 +1,20 @@
+/*
+All header options in TWS projects.
+
+Any instances of a header that involve greater specificity of styles beyond
+what is found in typography.scss should @extend from there.
+*/
+
+h1.dark {
+	@extend h1;
```

Comment:
Rule sets should be ordered as follows: `@extends`, `@includes` without `@content`, properties, `@includes` with `@content`, nested rule sets<br>Line should be indented with spaces, not tabs